app/models/user.js

```diff
@@ -0,0 +1,40 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('User', {
+    id: {
```

Comment:
This is default behavior of sequelize, remove and automatically Sequelize add tis